tpl/tplimpl/embedded/templates/shortcodes/youtube.html

```diff
@@ -4,7 +4,19 @@
 {{- $id := .Get "id" | default (.Get 0) -}}
 {{- $class := .Get "class" | default (.Get 1) -}}
 {{- $title := .Get "title" | default "YouTube Video" }}
+{{- $start := .Get "start" | default "" -}}
+{{- $autoplay := .Get "autoplay" | default "" -}}
+{{- $queryComponents := slice -}}
+{{- if $start -}}
+  {{- $queryComponents = $queryComponents | append "start" $start -}}
+{{- end -}}
+{{- if $autoplay -}}
+  {{- $queryComponents = $queryComponents | append "autoplay" 1 -}}
+{{- end -}}
+{{- $queryString := (querify $queryComponents | safeURL) | default "" -}}
+
```

Comment:
Could you remove this extra newline. There are some people that's a little bit pedantic about whitespace changes, even if it doesn't matter too much in the end.